pages/legal-and-technical-terms.md

```diff
@@ -0,0 +1,12 @@
+---
+permalink: /legal-and-technical-terms/
+title: Legal and technical terms
+---
+
+It's important that we present complicated information simply to increase the likelihood people will understand it.
```

Comment:
“simply to increase” reads strangely to me there because of its possible dual meaning.
I'm also tempted to substitute “clearly” for “simply”.

How about: “It's important to present complicated information clearly to make it easer to understand.”?